api/proto/config/devmodel.proto

```diff
@@ -56,3 +86,57 @@ message SystemAdapter {
 	// and used for app instances
 	string logicalName = 6;
 }
+
+enum PhyIoType {
+        PhyIoNoop = 0;
+        PhyIoEth = 1;
+        PhyIoUSB = 2;
+        PhyIoCOM = 3;
+        PhyIoAudio = 4;
+        PhyIoWLAN = 5;
```

Comment:
See comment I just make in other review.

Reply:
Fine with me, but then we should define the set which are networking (because networking can be used for systemAdapters and today we have specific checks for this in Eve by comparing with "CioEth". If such checks aren't needed on the cloud and UI, then we can keep the definition of the set on the Eve side. But I suspect the UI also check which things are ethernets/networks. Makes sense to check that with Sahil. Or we can defer adding the enums until after we check with him

Erik's comments ---- above

Reply:
I added your comment from other thread here. I agree we need other, we can add wrapper function in utils, which provide that checks are returns it is networking interfaces if it is either Eth/Wifi/Wlan etc.

I don't know how to do that in protobuf yet.

Reply:
One way to do this in protobuf+go is to define the enum values with a common prefix (e.g., PhyIoNet) so we get PhyIoNetEth, PhyIoNetWlan, PhyIoNetWwan) and then use the generated <type>_name map and doing prefix string matches against "PhyIoNet".

Alteratively we define a PhyIoNetMin and PhyIoNetMax value and put all of the networking values inbetween those two.

Reply:
Done, added PhyIoNet